NewKernel_d/include/CGAL/NewKernel_d/function_objects_cartesian.h

```diff
@@ -317,14 +318,14 @@ template<class R_> struct Contained_in_linear_hull : private Store_kernel<R_> {
 		// FIXME: Uh? Using it on a vector ?!
 		int d=vd(w);
 		Matrix m(d,n+1);
-		for(int i=0; f!=e; ++f,++i){
+		for(Index i=0; f!=e; ++f,++i){
 		  Vector const& v = *f;
-		  for(int j=0;j<d;++j){
-		    m(j,i)=c(v,j);
+		  for(Index j=0;j<d;++j){
+		    m(j,i)=c(v,static_cast<int>(j));
 		  }
 		}
-		for(int j=0;j<d;++j){
-		  m(j,n)=c(w,j);
+		for(Index j=0;j<d;++j){
+		  m(j,n)=c(w,static_cast<int>(j));
```

Comment:
Don't you also need `n` to be an Index then? It might have been easier just to make `n` an int, that might have been sufficient to work around the bug.